editor/components/inserter/group.js

```diff
@@ -74,6 +74,7 @@ export default class InserterGroup extends Component {
 				onMouseLeave={ this.createToggleBlockHover( null ) }
 				onFocus={ this.createToggleBlockHover( item ) }
 				onBlur={ this.createToggleBlockHover( null ) }
+				aria-label={ item.title } // Fix for IE11 and JAWS 2018 see GitHub issue 5827.
```

Comment:
Minor: Not certain we want to mention the GitHub issue since it's not clear what will happen post merge.